### PR TITLE
fsuppssind

### PR DESCRIPTION
**Pull request by icecream17**
moved <del>3</del> <ins>5</ins> theorems to main and shorten proofs automatically

otherwise mathbox

--comment
I was going to use fsuppssind to prove mhpmuldeg and mhppwdeg (currently commented out), but then I noticed df-mdeg's proofs, whose way of proving is probably better... So unfortunately, proving this theorem is mostly for nothing.

### CODE REVIEW

**Comment by icecream17** [comment]
> since I wont be able to edit this for a while and there will be merge conflicts, temporarily setting this to draft to give precedence to other prs

**edit**: resolved

**Comment by tirix** [comment]
The changes look fine to me, the errors come from missing `$d` disjoint variable restrictions.

**Review by tirix — APPROVED** [review]
This moves several "deduction mode" theorems to main, OK for me!

File: set.mm [hunk 1]
```diff
@@ -637993,6 +638049,158 @@ proof usage (see ~ mplelbas and ~ mpladd ).  (Contributed by SN,
       VCVVD $.
   $}
 
+  ${
+    $d ph x $.  $d F x $.  $d I x $.
+    fsuppssindlem1.z $e |- .0. = ( 0g ` G ) $.
```
Comment (avekens) [inline set.mm:638054]: What is ` G ` in this context? Is this hypothesis really necessary?

Comment (tirix) [reply set.mm:638054]: Indeed I guess here all that's needed is that `.0.` is a set, like in `.0. e. W`.

**Comment by tirix** [comment]
Are we good to merge this PR ? @avekens

**Review by avekens — APPROVED** [review]
Everything is fine now.

**Pull request merged**